README.md

```diff
@@ -98,8 +98,6 @@ code, other repos are also under active development:
   for Kong.
 - [Kong CloudFormation](https://github.com/Kong/kong-dist-cloudformation):
   Kong in a 1-click deployment for AWS EC2
-- [Kong AWS AMI](https://aws.amazon.com/marketplace/pp/B014GHERVU): Kong AMI on
```

Comment:
We should probably update the link to the new AMI instead: https://aws.amazon.com/marketplace/pp/B06WP4TNKL

Reply:
Oh, you are right of course. Didn't find it on my search